pom.xml

```diff
@@ -25,7 +19,10 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.build.resourceEncoding>UTF-8</project.build.resourceEncoding>
 
-        <byte-buddy.version>1.15.10</byte-buddy.version>
+        <!-- Maven minimum version -->
```

Comment:
Set default maven version 3.6.3 as maven has stated all older versions are obsolete.  This only matters for the build, nothing else, technically it should be 3.9.9 but keeping it low as not sure what other contributors are using currently.  This comes from the original oss parent that was dealing with maven 2 versions.